R/googleComputeEngine.R

```diff
@@ -0,0 +1,23 @@
+#' @export
+#' @importFrom googleComputeEngineR gce_ssh
+docker_cmd.gce_instance <- function(host, cmd = NULL, args = NULL,
```

Comment:
Oh wait, never mind. I guess it makes sense to keep this function in this package. I don't think you need the `importFrom` though, since you're using `googleComputeEngineR::gce_ssh`.